qiita_db/study.py

```diff
@@ -921,7 +957,46 @@ def ebi_submission_status(self, value):
 
     ebi_submission_status.__doc__.format(', '.join(_VALID_EBI_STATUS))
 
-    # --- methods ---
+    @property
+    def tags(self):
+        """Returns the tags of the study
+
+        Returns
+        -------
+        list of str
+            The study tags
+        """
+        with qdb.sql_connection.TRN:
+            sql = """SELECT study_tag_id, study_tag
+                        FROM qiita.study_tags
+                        LEFT JOIN qiita.per_study_tags USING (study_tag_id)
+                        WHERE study_id = {0}""".format(self._id)
+            qdb.sql_connection.TRN.add(sql)
+            return qdb.sql_connection.TRN.execute_fetchindex()
+
+    @tags.setter
+    def tags(self, tag_ids):
+        """Sets the tags of the study
+
+        Parameters
+        ----------
+        tag_ids : list of int
```

Comment:
Can we provide the list of tags rather than the list of tag ids? We have been removing the exposure of DB ids for these types of thins as much as we could, so I would like not to re-add some of them back